### PR TITLE
Prepare Webdev version `3.0.2` for release

### PR DESCRIPTION
**Pull request by elliette**
Work towards https://github.com/dart-lang/sdk/issues/51732

### CODE REVIEW

File: test_common/pubspec.yaml [hunk 1]
```diff
@@ -16,6 +15,6 @@ dependencies:
 dev_dependencies:
   lints: ^2.0.0
 
-dependency_overrides:
-  dwds:
-    path: ../dwds
+# dependency_overrides:
```
Comment (jakemac53) [inline test_common/pubspec.yaml:18]: This is really unlrelated, but we have been switching to use `pubspec_overrides.yaml` now - that way `pub lish` will not complain about the dependency overrides and you can stop having to toggle these on/off :).

See https://github.com/dart-lang/test/blob/master/pkgs/test/pubspec_overrides.yaml as an example

**Review by jakemac53 — APPROVED** [review]

File: test_common/pubspec.yaml (hunk 1 above)
Comment (elliette) [reply test_common/pubspec.yaml:18]: Oh that will be great! Opened https://github.com/dart-lang/webdev/issues/2040 as follow-up

Comment (annagrin) [reply test_common/pubspec.yaml:18]: Btw do we have to remove the overrides? How does this work with the local dwds that uses this project as well?

**Review by annagrin — APPROVED** [review]
LGTM with questions

File: test_common/pubspec.yaml (hunk 1 above)
Comment (annagrin) [reply test_common/pubspec.yaml:18]: I think I got it - local dwds currently has the same version. It's break once we update the version in the next PR but then we can just uncomment the overrides (until we start using the pubspec_overrides.yaml)

**Pull request merged**